### DISCUSSION TITLE
Is it possible to save my drawing to show the tree progressively folding as a GIF or video?

### DISCUSSION THREAD

**Discussion by McKlose03** [jgraph/drawio#4481] [Q&A]
Hey ! I need to display the flowchart as an animation; as gif, video or powerpoint. Is there any way to save my project as some of these extensions?

**Comment by davidjgraph** [comment]
There is no export to git or other kind of animation built-in. You'd have to use a third-party tool.

**Reply by McKlose03** [reply]
Do you know of any that you recommend?
